packages/docusaurus/src/commands/build.ts

```diff
@@ -224,7 +229,11 @@ async function buildLocale({
       if (!plugin.postBuild) {
         return;
       }
-      await plugin.postBuild({...props, content: plugin.content});
+      await plugin.postBuild({
+        ...props,
+        helmet: headTags,
```

Comment:
maybe rename to head or headTags?

Not fan of using "helmet" name here, lib is an implementation detail and we already use "head" name in other places

Also not fan of the API surface it introduces.

Users shouldn't use something like `helmet[route]?.meta.toComponent()`, this is a quite confusing that we'd struggle to document well and ensure retrocompatibility on.

Maybe we could only expose a few route attributes and transform all `HelmetDatum` types to a convenient format, easy to document?